src/services/videoconference/logic/index.js

```diff
@@ -0,0 +1,102 @@
+const rp = require('request-promise-native');
+const xml2js = require('xml2js-es6-promise');
+const { error } = require('../../../logger');
+const {
+	MESSAGE_KEYS, RETURN_CODES, ROLES, GUEST_POLICIES,
+} = require('./constants');
+
+const createParams = { allowStartStopRecording: false, guestPolicy: GUEST_POLICIES.ALWAYS_DENY };
+
+/**
+ * responseheaders will be evaluated to have a set-cookie valuecontaining a jsessionid
+ * @param {*} headers
+ */
+const getSessionCookieFromHeaders = (headers) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['getSessionCookieFromHeaders' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4819756)